### ISSUE TITLE
Controller Support

### ISSUE THREAD

**Issue by sduensin** [mattiasgustavsson/dos-like#47]
App.h and dos-like would really benefit from gamepad/joystick support!

**Comment by mattiasgustavsson** [comment]
I agree, would love to have controller support. But I don’t know web programming very well, and I don’t know how to do gamepad input in webassembly. I hope to get around to learn it some day, so I can add it to app.h. Or maybe someone else will contribute a PR for it 🙂
